client-src/elements/chromedash-app.js

```diff
@@ -169,7 +169,7 @@ class ChromedashApp extends LitElement {
       this.pageComponent.contextLink = this.contextLink;
       this.currentPage = ctx.path;
       this.pageComponent.appTitle = this.appTitle;
-      if (this.pageComponent.featureId != this.gateColumnRef.value.feature?.id) {
+      if (this.pageComponent.featureId != this.gateColumnRef.value?.feature?.id) {
```

Comment:
I was occasionally getting some errors when reloading the page until making this change.

Reply:
Sorry about that.  I have the same fix in my current PR.